pkg/policy/selectorcache.go

```diff
@@ -238,7 +302,13 @@ type labelIdentitySelector struct {
 	namespaces []string // allowed namespaces, or nil
 }
 
-func (l *labelIdentitySelector) MatchesNamespace(ns string) bool {
+// xxxMatches returns true if the CachedSelector matches given labels.
+// This is slow, but only used for policy tracing, so it's OK.
+func (l *labelIdentitySelector) xxxMatches(labels labels.LabelArray) bool {
```

Comment:
Why `xxx`?

Reply:
To remind that this is not a cached match. Maybe "SlowMatches" would be better?